src/iotjs_util.c

```diff
@@ -27,7 +27,10 @@
 
 iotjs_string_t iotjs_file_read(const char* path) {
   FILE* file = fopen(path, "rb");
-  IOTJS_ASSERT(file != NULL);
+  if (file == NULL) {
+    iotjs_string_t empty_content = iotjs_string_create();
+    return empty_content;
+  }
```

Comment:
@glistening I found a test for this, test/run_pass/test_process_readsource.js. I am not sure that we need to distinguish empty content and fopen error. process.readsource is not a official API and it's only case that is used..